internal/pkg/call/call_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package call
+
+import (
+	"fmt"
+	"go/ast"
+	"go/importer"
+	"go/parser"
+	"go/token"
+	"go/types"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"golang.org/x/tools/go/ssa"
+	"golang.org/x/tools/go/ssa/ssautil"
+)
+
+type fakeReferrer struct {
+	hasPathTo bool
+}
+
+func (f fakeReferrer) RefersTo(node ssa.Node) bool { return f.hasPathTo }
+
+func TestRegularCallReferredBy(t *testing.T) {
+	source := readFromTestData(t, "test.go")
+	cases := []struct {
+		r              Referrer
+		parentFuncName string
+		want           bool
+		desc           string
+	}{
+		{fakeReferrer{true}, "CallNoArgs", false, "call without args"},
+		{fakeReferrer{true}, "CallOneArg", true, "call with one arg that is referred to by a referrer"},
+		{fakeReferrer{false}, "CallOneArg", false, "call with one arg that isn't referred to"},
+	}
+	for _, tt := range cases {
+		t.Run(tt.desc, func(t *testing.T) {
+			ssaCall := getCall(t, source, tt.parentFuncName)
+			call := Regular(ssaCall)
+			got := call.ReferredBy(tt.r)
+			if got != tt.want {
+				t.Errorf("call.ReferredBy(%v) == %v, want %v", call, got, tt.want)
+			}
+		})
+	}
+}
+
+func readFromTestData(t *testing.T, filename string) string {
```

Comment:
This would also go away if you were to use analysistest.
`dir := analysistest.TestData()`